Project.toml

```diff
@@ -1,29 +1,20 @@
-name = "REALMSupport"
-uuid = "926d8977-77e3-4bba-bc27-2e234c0c0038"
-authors = ["Jonathan Chang <jona125@gmail.com> and contributors"]
-version = "0.1.0"
-
 [deps]
+CoordinateTransformations = "150eb455-5306-5404-9cee-2592286d6298"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 ImageSegmentation = "80713f31-8817-5129-9cf8-209ff8fb23e1"
 ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
 ImageView = "86fae568-95e7-573e-a6b2-d8a6b900c9ef"
 Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
 ImagineFormat = "4bab44a2-5ff2-5a6b-8e10-825fb9ac126a"
+Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
 Makie = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
 OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
 Optim = "429524aa-4258-5aef-a3af-852621145aeb"
+PaddedViews = "5432bcbf-9aad-5242-b902-cca2824c8663"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
+RegisterMismatch = "3c0dd727-6833-5f5d-a1e8-c0d421935c74"
+Rotations = "6038ab10-8711-5258-84ad-4b1120ba62dc"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
-
-[compat]
-julia = "1"
-
-[extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-
-[targets]
-test = ["Test"]
```

Comment:
I can help get these updated, I have a PR ready but perhaps we should get this PR merged first.